modules/appeals_api/app/swagger/v0/appeals.yml

```diff
@@ -3,38 +3,33 @@ info:
   version: 0.0.1
   title: Appeals
   description: |
-    Veterans Benefits Administration (VBA) caseflow appeals status.
+    The Appeals Status API allows you to request the statuses of all decision reviews for a Veteran, including decision reviews following the AMA process and legacy benefit appeals. The statuses are returned as read only. If you need to manage or submit a Veteran’s decision review request, use the [Decision Reviews API](/explore/appeals/docs/decision_reviews?version=current). 
 
     ## Background
 
-    Use this API to request the status of a Veteran's benefits claim appeal.
-
-    The Appeals API passes data through to Caseflow, a case management system. Caseflow
-    accepts a header with the Veteran’s SSN and returns the current status of their appeals.
+    The Appeals API passes data through to Caseflow, a case management system. Caseflow accepts a header with the Veteran’s SSN and returns the current status of their decision reviews and/or benefits appeals.
 
     Because this application is designed to allow third-parties to request information on behalf of a Veteran, we are not using VA Authentication Federation Infrastructure (VAAFI) headers or Single Sign On External (SSOe).
 
+
     ## Design
 
-    ### Authorization
+    ### Authorization and Access
 
-    API requests are authorized through a symmetric API token, provided in an HTTP header with name "apikey".
+    To gain access to the Appeals Status API you must [request an API Key](/apply). API requests are authorized through a symmetric API token which is provided in an HTTP header named `apikey`.
 
-    ### Status Request
+    ### Data refresh
 
-    Allows a client to check benefit appeals status
+    The Lighthouse Sandbox environment points to the Caseflow UAT environment which refreshes daily. The data in this environment is mock data and does not contain any PII.
```

Comment:
UAT doesn't refresh daily
https://dsva.slack.com/archives/C2ZAMLK88/p1596053539041000?thread_ts=1596053120.040900&cid=C2ZAMLK88

Reply:
confirmed that UAT's DB is never reset
https://dsva.slack.com/archives/C2ZAMLK88/p1596123119046300?thread_ts=1596053120.040900&cid=C2ZAMLK88